docs/website/docs/overview/dev_and_deploy.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Develop and Deploy
+sidebar_position: 5
+---
+
+# Develop and Deploy
+
+The two most important commands in `odo` are `odo dev` and `odo deploy`. 
+
+In some situations, you'd want to use [`odo dev`](/docs/command-reference/dev) over [`odo deploy`](/docs/command-reference/deploy) and vice-versa. This document highlights when you should use either command.
+
+## When should I use `odo dev`?
+
+`odo dev` should be used in the initial development process of your application. 
+
+For example, you should use `odo dev` when you are:
```

Comment:
```suggestion
For example, you should use `odo dev` when you are working with a local development environment and:
```

Reply:
The development environment is not necessarily local, no? I think we should just mention a "development environment".